### ISSUE TITLE
FF3-1

### ISSUE THREAD

**Issue by bdw666** [capitalone/fpe#30]
Is there a plan to update the FF3 code to FF3-1? Thanks!

**Comment by anitgandhi** [comment]
Hello!

I've been waiting on NIST to finalize Revision 1. It seems like it's been in draft for a while.

**Comment by bdw666** [comment]
yeah, it's been over a year :) Hashicorp vault has implemented the draft, but I love you code and would like to use it :) I'm building a system that is resilient to fpe algorithm change. thanks for the quick response!
